build/openresty/BUILD.openresty.bazel

```diff
@@ -218,10 +295,41 @@ configure_make(
         "@lua-resty-lmdb//:all_srcs",
         "@lua-resty-events//:all_srcs",
         "@openresty_binding//:all_srcs",
+        "@ngx_wasm_module//:all_srcs",
+        "@v8//:all_srcs",
+        "@wasmer//:all_srcs",
+        "@wasmtime//:all_srcs",
+        "@openresty//:wasmx_v8_ar",
     ],
     configure_command = "configure",
     configure_in_place = True,
     configure_options = CONFIGURE_OPTIONS,
+    env = select({
+        "@kong//:wasmx_v8": {
+            "NGX_WASM_RUNTIME": "v8",
+            "NGX_WASM_RUNTIME_LIB": "$$EXT_BUILD_ROOT$$/external/v8/lib",
+            "NGX_WASM_RUNTIME_INC": "$$EXT_BUILD_ROOT$$/external/v8/include",
+            # https://github.com/Kong/ngx_wasm_module/blob/0f07c712c48d410190ec5e0cc0b34fdfd190387d/t/10-build/003-dynamic_module.t#L43
+            "NGX_WASM_RUNTIME_LD_OPT": "$$EXT_BUILD_ROOT$$/external/v8/lib/libwee8.a -lv8bridge -lstdc++ -lm -ldl -lpthread",
```

Comment:
Hmmm the build do not need these `LD_OPT` environment variables, they are designed to work without having to specify these options. I just tested removing these linker options locally and it worked. This is how the build step is documented in our [INSTALL.md](https://github.com/Kong/ngx_wasm_module/blob/main/docs/INSTALL.md). This is because these flags are already added by our `config` script, and then appended to Nginx build flags belonging to the runtime [here](https://github.com/Kong/ngx_wasm_module/blob/main/config#L87).

These tests in the linked suite should probably be refactored to reflect that.

Reply:
Oh ok I think I understand better post-lunch haha. These are for the static linking part. I think they should move to the above `--with-ld-opt=` flags, indicating that Kong's Bazel build system follows ngx_wasm_module's standard INSTALL.md compilation method: `./configure` flags and an `NGX_WASM_RUNTIME` env variable.

Reply:
I think the only difference with my initial comment is that only `$$EXT_BUILD_ROOT$$/external/v8/lib/libwee8.a` should be necessary, `-lv8bridge -lstdc++ -lm -ldl -lpthread` are all automatically added already. For statically linking a runtime, only `--with-ld-opt=/path/to/runtime.a` should be needed. At least that's the expectations.

Having these options duplicated won't be helpful in the future, I think we should remove them even if a subsequent PR.

Reply:
Thanks for drawing attention to this. There was a lot of throwing things against the wall to see what stuck whilst working on the build/CI integration, so it's unsurprising to me that there is some unnecessary cruft in here.